src/tactic/squeeze.lean

```diff
@@ -284,12 +293,30 @@ do (cfg',c) ← parse_config cfg,
           sformat!"Try this: simpa{use_iota_eqn} only "
           sformat!"{attrs}{tgt'}{c}" args)
 
+/-- see `squeeze_simp` -/
+meta def squeeze_dsimp
```

Comment:
Cool! Maybe you could split this into a separate PR?

Reply:
I could but I think this PR is pretty much ready. Is it worth splitting it before merging?

Reply:
@gebner Do you think a lot of work is needed before merging this PR?

Reply:
I think it would be neater to have `squeeze_dsimp` in a separate PR, but I won't insist on it.

Reply:
> I think it would be neater to have `squeeze_dsimp` in a separate PR, but I won't insist on it.

I agree, if only for the commit message in the git history.

Reply:
Done: #3386